conf/nomad/minio.hcl

```
@@ -52,7 +52,16 @@ job "${service_name}" {
         timeout   = "4s"
       }
       connect {
-        sidecar_service {}
+        sidecar_service {
+ %{ for upstream in jsondecode(upstreams) }
+          proxy {
+            upstreams {
+              destination_name = "${upstream.service_name}"
+              local_bind_port = "${upstream.port}"
+            }
+          }
+%{ endfor }
```

It's only upstreams stanza that needs to be looped, isn't it? https://github.com/fredrikhgrelland/terraform-nomad-presto/blob/c1a54347b1689a6234067db2245e8a1ff71697fe/conf/nomad/presto.hcl#L33-L49
```suggestion
          proxy {
 %{ for upstream in jsondecode(upstreams) }
            upstreams {
              destination_name = "${upstream.service_name}"
              local_bind_port = "${upstream.port}"
            }
%{ endfor }
          }
```

---

Yes, good catch. Ty!